carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/locator/ExtendedElementLocator.java

```diff
@@ -187,13 +187,12 @@ private WebElement findElementByAI(WebDriver drv, Label label, String caption) {
      */
     public static By toCaseInsensitive(String locator) {
         locator = StringUtils.remove(locator, "By.xpath: ");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "locator". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1226)